hpy/devel/include/hpy/inline_helpers.h

```diff
@@ -459,4 +459,30 @@ HPySlice_AdjustIndices(HPyContext *_HPy_UNUSED_ARG(ctx), HPy_ssize_t length,
     return 0;
 }
 
+HPyAPI_INLINE_HELPER HPy
+HPy_CallMethodTupleDict_s(HPyContext *ctx, HPy receiver, const char *name, HPy args, HPy kw)
```

Comment:
I wonder whether we should swap `receiver` and `name` here, to better match the signature in `HPy_CallMethod`

Reply:
good point; I'll do that